core/trino-main/src/main/java/io/trino/memory/LocalMemoryManager.java

```diff
@@ -63,7 +65,7 @@ private void validateHeapHeadroom(NodeMemoryConfig config, long availableMemory)
 
     public MemoryInfo getInfo()
     {
-        return new MemoryInfo(AVAILABLE_PROCESSORS.get(), memoryPool.getInfo());
+        return new MemoryInfo(AVAILABLE_PROCESSORS.get(), ((UnixOperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean()).getCpuLoad(), memoryPool.getInfo());
```

Comment:
I think that we should memoize with timeout system load (for 5 seconds)?

Reply:
Is there a reason why we should memoize this? I'm not an expert on how this library is implemented, but I'd guess it's trivial to retrieve this information. Also, I don't anticipate this endpoint being spammed in a short amount of time since this is mainly used by Gateway. So unless the user has 10k Gateways running at the same time and each of them calls this endpoint every 500ms, in which case I'd argue they should cache it on the gateway side, not trino, it should be a low-effort retrieval. WDYT?